lib/utils/constants.js

```diff
@@ -560,9 +562,11 @@ export default {
     'H70BC', // https://app-h5.govee.com/user-manual/wlan-guide (2024-10-15)
     'H70C1', // https://app-h5.govee.com/user-manual/wlan-guide (2024-10-15)
     'H70C2', // https://app-h5.govee.com/user-manual/wlan-guide (2024-10-15)
+    'H8072', // https://app-h5.govee.com/user-manual/wlan-guide (2024-10-15)
```

Comment:
are these two new models LAN compatible? i could not see them in the list in the wlan-guide but it sometimes gets out of date.

Reply:
Yes, both devices support LAN control